src/shared/vscode/user_prompts.ts

```diff
@@ -1,25 +1,25 @@
 import * as fs from "fs";
 import * as os from "os";
 import * as path from "path";
-import { Context } from "vm";
 import * as vs from "vscode";
-import { doNotAskAgainAction, flutterSurvey2019Q2PromptWithAnalytics, flutterSurvey2019Q2PromptWithoutAnalytics, isWin, longRepeatPromptThreshold, noRepeatPromptThreshold, noThanksAction, openDevToolsAction, takeSurveyAction, wantToTryDevToolsPrompt } from "../constants";
+import { doNotAskAgainAction, flutterSurveyPromptWithAnalytics, flutterSurveyPromptWithoutAnalytics, isWin, longRepeatPromptThreshold, noRepeatPromptThreshold, noThanksAction, openDevToolsAction, takeSurveyAction, wantToTryDevToolsPrompt } from "../constants";
 import { Logger } from "../interfaces";
 import { openInBrowser } from "./utils";
+import { Context } from "./workspace";
 
-// Mon May 13 2019 20:00:00 GMT+0100 (BST) = noon PDT on 13th May
-export const surveyStart = Date.UTC(2019, 4 /* Month is 0-based!! */, 13, 19, 0);
-// Mon May 27 2019 08:00:00 GMT+0100 (BST) = midnight PDT between 26th/27th may.
-export const surveyEnd = Date.UTC(2019, 4 /* Month is 0-based!! */, 27, 7, 0);
+// Mon Aug 12 2019 17:00:00 GMT+0100 (British Summer Time UTC+1) = Mon Aug 12 09:00 PDT (UTC-7)
+export const surveyStart = Date.UTC(2019, 7 /* Month is 0-based!! */, 12, 16, 0);
+// Sun Aug 25 2019 02:00:00 GMT+0100 (British Summer Time UTC+1) = Sat Aug 24 18:00 PDT (UTC-7).
+export const surveyEnd = Date.UTC(2019, 7 /* Month is 0-based!! */, 25, 1, 0);
```

Comment:
@jayoung-lee please double-check these dates. They're written as UTC and to confuse things, months are 0-based and the end date is "the next  day" in UTC.

The first parts of the comments (the time written in BST) are the output of printing the date from the next line in a JS console (though the conversions after the `=` are my own!).

Reply:
LGTM :)

Reply:
Thanks!